packages/manager/apps/web/client/app/private-database/configuration/private-database-configuration.controller.js

```diff
@@ -83,6 +83,21 @@ angular.module('App').controller(
           };
         });
 
+        field.selectedValue = find(field.availableValues, {
+          id: field.value,
+        });
```

Comment:
we avoid to use external functions if native one could do it.
```suggestion
        field.selectedValue = field.availableValues.find((value) => value.id === field.value);
```

Reply:
Suggestion applied on whole file.